src/compas_ags/rhino/forceartist.py

```diff
@@ -59,7 +59,7 @@ def draw(self):
             color = {}
             color.update({vertex: self.settings['color.vertices'] for vertex in self.diagram.vertices()})
             color.update({vertex: self.settings['color.vertices:is_fixed'] for vertex in self.diagram.vertices_where({'is_fixed': True})})
-            color[self.anchor_vertex] = self.settings['color.anchor']
+            # color[self.anchor_vertex] = self.settings['color.anchor']
```

Comment:
why not remove the line?

Reply:
it is commented out because we don't want to highlight the anchor vertex anymore?

Reply:
yes... just don't hightlight the anchor vertex

Reply:
okay but then indeed remove the comments...